explainer.md

```diff
@@ -478,60 +478,59 @@ function onDrawFrame() {
 }
 ```
 
-### Presenting automatically when the user interacts with the headset
+### Responding to a reset pose
```

Comment:
Sorry, I moved the automatic presentation section below this one to help it flow better with the navigation section, but it's made the diff kind of ugly to read. The entire "Responding to a reset pose" section is unchanged.